### ISSUE TITLE
RB260GS OS detection

### ISSUE THREAD

**Issue by MLWALK3R** [librenms/librenms#1543]
RouterBoard device not showing logo or device type 

![RouterBoard](http://lel.ovh/1438087215)

**Comment by Rosiak** [comment]
Can you please post output from an snmpwalk?

**Comment by MLWALK3R** [comment]
```bash
iso.3.6.1.2.1.1.1.0 = STRING: "RB260GS"
iso.3.6.1.2.1.1.2.0 = OID: iso.3.6.1.4.1.14988.2
iso.3.6.1.2.1.1.3.0 = Timeticks: (1549880351) 179 days, 9:13:23.51
iso.3.6.1.2.1.1.4.0 = STRING: "Michael Walker"
iso.3.6.1.2.1.1.5.0 = STRING: "MikroTik"
iso.3.6.1.2.1.1.6.0 = STRING: "Widnes"
iso.3.6.1.2.1.1.7.0 = INTEGER: 2
iso.3.6.1.2.1.2.1.0 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.1.1 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.1.2 = INTEGER: 2
iso.3.6.1.2.1.2.2.1.1.3 = INTEGER: 3
iso.3.6.1.2.1.2.2.1.1.4 = INTEGER: 4
iso.3.6.1.2.1.2.2.1.1.5 = INTEGER: 5
iso.3.6.1.2.1.2.2.1.1.6 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.2.1 = STRING: "Port1"
iso.3.6.1.2.1.2.2.1.2.2 = STRING: "Port2"
iso.3.6.1.2.1.2.2.1.2.3 = STRING: "Port3"
iso.3.6.1.2.1.2.2.1.2.4 = STRING: "Port4"
iso.3.6.1.2.1.2.2.1.2.5 = STRING: "Port5"
iso.3.6.1.2.1.2.2.1.2.6 = STRING: "SFP"
iso.3.6.1.2.1.2.2.1.3.1 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.3.2 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.3.3 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.3.4 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.3.5 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.3.6 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.4.1 = INTEGER: 1500
iso.3.6.1.2.1.2.2.1.4.2 = INTEGER: 1500
iso.3.6.1.2.1.2.2.1.4.3 = INTEGER: 1500
iso.3.6.1.2.1.2.2.1.4.4 = INTEGER: 1500
iso.3.6.1.2.1.2.2.1.4.5 = INTEGER: 1500
iso.3.6.1.2.1.2.2.1.4.6 = INTEGER: 1500
iso.3.6.1.2.1.2.2.1.5.1 = Gauge32: 1000000000
iso.3.6.1.2.1.2.2.1.5.2 = Gauge32: 1000000000
iso.3.6.1.2.1.2.2.1.5.3 = Gauge32: 1000000000
iso.3.6.1.2.1.2.2.1.5.4 = Gauge32: 100000000
iso.3.6.1.2.1.2.2.1.5.5 = Gauge32: 100000000
iso.3.6.1.2.1.2.2.1.5.6 = Gauge32: 1000000000
iso.3.6.1.2.1.2.2.1.6.1 = Hex-STRING: 00 0C 42 7F B7 E2
iso.3.6.1.2.1.2.2.1.6.2 = Hex-STRING: 00 0C 42 7F B7 E2
iso.3.6.1.2.1.2.2.1.6.3 = Hex-STRING: 00 0C 42 7F B7 E2
iso.3.6.1.2.1.2.2.1.6.4 = Hex-STRING: 00 0C 42 7F B7 E2
iso.3.6.1.2.1.2.2.1.6.5 = Hex-STRING: 00 0C 42 7F B7 E2
iso.3.6.1.2.1.2.2.1.6.6 = Hex-STRING: 00 0C 42 7F B7 E2
iso.3.6.1.2.1.2.2.1.7.1 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.7.2 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.7.3 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.7.4 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.7.5 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.7.6 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.8.1 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.8.2 = INTEGER: 2
iso.3.6.1.2.1.2.2.1.8.3 = INTEGER: 2
iso.3.6.1.2.1.2.2.1.8.4 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.8.5 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.8.6 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.9.1 = Timeticks: (1539149956) 178 days, 3:24:59.56
iso.3.6.1.2.1.2.2.1.9.2 = Timeticks: (607482739) 70 days, 7:27:07.39
iso.3.6.1.2.1.2.2.1.9.3 = Timeticks: (1532900485) 177 days, 10:03:24.85
iso.3.6.1.2.1.2.2.1.9.4 = Timeticks: (1492469922) 172 days, 17:44:59.22
iso.3.6.1.2.1.2.2.1.9.5 = Timeticks: (1482075176) 171 days, 12:52:31.76
iso.3.6.1.2.1.2.2.1.9.6 = Timeticks: (0) 0:00:00.00
iso.3.6.1.2.1.2.2.1.10.1 = Counter32: 1438240366
iso.3.6.1.2.1.2.2.1.10.2 = Counter32: 4410950
iso.3.6.1.2.1.2.2.1.10.3 = Counter32: 71940
iso.3.6.1.2.1.2.2.1.10.4 = Counter32: 1121480439
iso.3.6.1.2.1.2.2.1.10.5 = Counter32: 527826210
iso.3.6.1.2.1.2.2.1.10.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.11.1 = Counter32: 424640317
iso.3.6.1.2.1.2.2.1.11.2 = Counter32: 48000
iso.3.6.1.2.1.2.2.1.11.3 = Counter32: 152
iso.3.6.1.2.1.2.2.1.11.4 = Counter32: 163846987
iso.3.6.1.2.1.2.2.1.11.5 = Counter32: 50910034
iso.3.6.1.2.1.2.2.1.11.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.12.1 = Counter32: 34424148
iso.3.6.1.2.1.2.2.1.12.2 = Counter32: 270
iso.3.6.1.2.1.2.2.1.12.3 = Counter32: 426
iso.3.6.1.2.1.2.2.1.12.4 = Counter32: 8890408
iso.3.6.1.2.1.2.2.1.12.5 = Counter32: 70424
iso.3.6.1.2.1.2.2.1.12.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.1 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.5 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.1 = Counter32: 1
iso.3.6.1.2.1.2.2.1.14.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.5 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.15.1 = Counter32: 0
iso.3.6.1.2.1.2.2.1.15.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.15.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.15.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.15.5 = Counter32: 0
iso.3.6.1.2.1.2.2.1.15.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.16.1 = Counter32: 2102903558
iso.3.6.1.2.1.2.2.1.16.2 = Counter32: 119789357
iso.3.6.1.2.1.2.2.1.16.3 = Counter32: 11788555
iso.3.6.1.2.1.2.2.1.16.4 = Counter32: 1054727306
iso.3.6.1.2.1.2.2.1.16.5 = Counter32: 4075538342
iso.3.6.1.2.1.2.2.1.16.6 = Counter32: 2675124660
iso.3.6.1.2.1.2.2.1.17.1 = Counter32: 219303528
iso.3.6.1.2.1.2.2.1.17.2 = Counter32: 81840
iso.3.6.1.2.1.2.2.1.17.3 = Counter32: 141
iso.3.6.1.2.1.2.2.1.17.4 = Counter32: 298989976
iso.3.6.1.2.1.2.2.1.17.5 = Counter32: 119698332
iso.3.6.1.2.1.2.2.1.17.6 = Counter32: 11167
iso.3.6.1.2.1.2.2.1.18.1 = Counter32: 9219463
iso.3.6.1.2.1.2.2.1.18.2 = Counter32: 9309
iso.3.6.1.2.1.2.2.1.18.3 = Counter32: 43552
iso.3.6.1.2.1.2.2.1.18.4 = Counter32: 34681811
iso.3.6.1.2.1.2.2.1.18.5 = Counter32: 40764602
iso.3.6.1.2.1.2.2.1.18.6 = Counter32: 43643991
iso.3.6.1.2.1.2.2.1.19.1 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.5 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.1 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.5 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.6 = Counter32: 0
End of MIB

```bash

**Comment by Rosiak** [comment]
cd /opt/librenms
git remote add rosiak https://github.com/rosiak/librenms-1
git remote update rosiak
git checkout issue-1543

Can you please test that?

**Comment by f0o** [comment]
Merged with #1545
